### ISSUE TITLE
Keep Ocim in one repo?

### ISSUE THREAD

**Issue by antoviaque** [open-craft/ocim-frontend#1]
@xitij2000 What's the reason for creating a separate repo for the Ocim frontend? Usually it makes it easier to manage to have all relevant code in a single repository - versions are easier to manage. 

CC @bradenmacdonald

**Comment by bradenmacdonald** [comment]
+1 I much prefer a single repo; just have a `backend` and `frontend` folder. It keeps them separated but makes management a lot easier, as a single PR can change the backend and frontend together.

**Comment by xitij2000** [comment]
@antoviaque Sure, I will move it to the main OCIM repository. 
@bradenmacdonald  I'm not sure how easy doing that will be at this stage. I will try it but doing so will break all existing PRs, and deployment scripts etc. I don't think we've budgeted for such a refactoring.

**Comment by bradenmacdonald** [comment]
@xitij2000 What do you mean refactoring?  This is a brand new repo. If you mean moving the backend into a `backend` folder, that's not necessary, but you can still move the frontend into a `frontend` folder :)

**Comment by xitij2000** [comment]
@bradenmacdonald  Yup, that's what I meant, moving all backend code to a backend folder. It would be nice but not sure how easy it'd be to accomplish now.

I've gone with just moving the frontend bits to a frontend folder. https://github.com/open-craft/opencraft/pull/484
